src/main/java/org/cyclops/integrateddynamics/api/network/IPositionedAddonsNetwork.java

```diff
@@ -51,11 +51,12 @@
 
     public static class PrioritizedPartPos implements Comparable<PrioritizedPartPos> {
         private final PartPos partPos;
-        private final int priority;
+        private final int priority, channel;
```

Comment:
The `compareTo` should also be extended to include this channel field, probably after the positions are compared. Even though it is not needed at the moment, in the future it may be possible to have multiple parts on the same positions, but on different channels.